docker_scripts/install_mimic.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+cd /
+git clone https://github.com/MycroftAI/mimic1.git
+cd mimic1
+./dependencies.sh --prefix="/usr/local"
+./autogen.sh
+./configure --prefix="/usr/local"
+make
+make check
```

Comment:
No need for this script on 16.04, correct? Anyway, if we keep it, we should change it to build a specific tag of mimic1, not the master branch.

Reply:
Yeah, no need. Included just in case we need to consider support for more versions of ubuntu down the line. But can remove if you think that's better